mods-install/install_sqlsrv.sh

```diff
@@ -8,9 +8,7 @@ cd tmp
 curl -L https://github.com/microsoft/msphpsql/releases/download/v5.9.0/Ubuntu2004-7.3.tar | tar xf - --strip-components=1 Ubuntu2004-7.3/php_pdo_sqlsrv_73_nts.so Ubuntu2004-7.3/php_sqlsrv_73_nts.so
 curl -L https://github.com/microsoft/msphpsql/releases/download/v5.10.1/Ubuntu2004-7.4.tar | tar xf - --strip-components=1 Ubuntu2004-7.4/php_pdo_sqlsrv_74_nts.so Ubuntu2004-7.4/php_sqlsrv_74_nts.so
 curl -L https://github.com/microsoft/msphpsql/releases/download/v5.11.1/Ubuntu2204-8.0.tar | tar xf - --strip-components=1 Ubuntu2204-8.0/php_pdo_sqlsrv_80_nts.so Ubuntu2204-8.0/php_sqlsrv_80_nts.so
-curl -L https://github.com/microsoft/msphpsql/releases/download/v5.12.0/Ubuntu2204-8.1.tar | tar xf - --strip-components=1 Ubuntu2204-8.1/php_pdo_sqlsrv_81_nts.so Ubuntu2204-8.1/php_sqlsrv_81_nts.so
-curl -L https://github.com/microsoft/msphpsql/releases/download/v5.12.0/Ubuntu2204-8.2.tar | tar xf - --strip-components=1 Ubuntu2204-8.2/php_pdo_sqlsrv_82_nts.so Ubuntu2204-8.2/php_sqlsrv_82_nts.so
-curl -L https://github.com/microsoft/msphpsql/releases/download/v5.12.0/Ubuntu2204-8.3.tar | tar xf - --strip-components=1 Ubuntu2204-8.3/php_pdo_sqlsrv_83_nts.so Ubuntu2204-8.3/php_sqlsrv_83_nts.so
+curl -L https://github.com/microsoft/msphpsql/releases/download/v5.12.0/Linux_5.12.0RTW.tar.gz | tar xzf -
```

Comment:
archive structure includes . as leading path component. Weird stuff. 

Extract with

```suggestion
curl -L https://github.com/microsoft/msphpsql/releases/download/v5.12.0/Linux_5.12.0RTW.tar.gz | tar xzf - --strip-components=2 \
    ./Linux/PDO_SQLSRV_8.1_NTS.so ./Linux/SQLSRV_8.1_NTS.so \
    ./Linux/PDO_SQLSRV_8.2_NTS.so ./Linux/SQLSRV_8.2_NTS.so \
    ./Linux/PDO_SQLSRV_8.3_NTS.so ./Linux/SQLSRV_8.3_NTS.so
```